app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -236,23 +244,50 @@ class AppVersionActivityTest {
   @Test
   fun testAppVersionActivity_loadFragment_onBackPressed_displaysAdministratorControlsActivity() {
     ActivityScenario.launch<AdministratorControlsActivity>(
-      launchAdministratorControlsActivityIntent(
-        internalProfileId = 0
-      )
-    ).use {
+      launchAdministratorControlsActivityIntent(internalProfileId = 0)
+    ).use { scenario ->
+
       testCoroutineDispatchers.runCurrent()
+
+      // Scroll to the item in the RecyclerView
```

Comment:
Comments are not required since ViewActions and Matchers are pretty self explanatory. We only add comments whne it is not clear some code has been added.